src/main/kotlin/no/nav/syfo/behandlendeenhet/kafka/BehandlendeEnhetProducer.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.syfo.behandlendeenhet.kafka
+
+import no.nav.syfo.behandlendeenhet.database.domain.PPerson
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+
+class BehandlendeEnhetProducer(
+    private val kafkaProducerBehandlendeEnhet: KafkaProducer<String, KBehandlendeEnhetUpdate>,
+) {
+    fun updateBehandlendeEnhet(
+        pPerson: PPerson,
+    ) {
+        try {
+            kafkaProducerBehandlendeEnhet.send(
+                ProducerRecord(
+                    BEHANDLENDE_ENHET_UPDATE_TOPIC,
+                    pPerson.personident,
+                    KBehandlendeEnhetUpdate(
+                        pPerson.personident,
+                        pPerson.updatedAt.toLocalDateTime(),
+                    ),
```

Comment:
Hva har vi pleid å gjøre her? Er det nødvendig å sende med personident i både key og value?

Reply:
Tror vi har gjort litt begge deler. I siste topic vi lagde på isdialogmote så var personident kun med som key til record, og ikke gjentatt i selve record. Det ble litt rart når vi skulle lese fra topic, for id hører naturlig til sammen med den andre informasjonen i recorden. Jeg tror jeg har landa på at den kan gjentas.

Reply:
Yes, vi kjører på med det 👌🏼